src/main/java/cubicchunks/world/column/Column.java

```diff
@@ -131,7 +126,7 @@ public int getHeightValue(int localX, int localZ) {
 
 		Integer skylightBlockY = getHeightmapAt(localX, localZ);
 		if (skylightBlockY == null) {
-			// PANIC!
+			// PANIC! TODO: Test this some more, and find solutions
```

Comment:
I am not changing things much in this PR... but I have already made some changes to that one, while refectoring.